stylize.py

```diff
@@ -8,14 +8,18 @@
 CONTENT_LAYER = 'relu4_2'
 STYLE_LAYERS = ('relu1_1', 'relu2_1', 'relu3_1', 'relu4_1', 'relu5_1')
 
-
-def stylize(network, initial, content, style, iterations,
-        content_weight, style_weight, tv_weight,
+def stylize(network, initial, content, styles, iterations,
+        content_weight, style_weight, style_blend_weights, tv_weight,
         learning_rate, print_iterations=None, checkpoint_iterations=None):
     shape = (1,) + content.shape
-    style_shape = (1,) + style.shape
+    style_shapes = []
+    num_styles = len(styles)
+    for i in range(0,num_styles):
```

Comment:
Don't need the lower bound for the `range()` constructor.
Also, it's more Pythonic to iterate over the objects:

```python
for style in styles:
    style_shapes.append((1,) + style.shape)
```

Or even more concise:

```python
style_shapes = [(1,) + style.shape for style in styles]
```